packages/react-docsite-components/src/components/PageHeader/PageHeader.tsx

```diff
@@ -1,23 +1,42 @@
 import * as React from 'react';
 import { IPageHeaderProps, IPageHeaderStyleProps, IPageHeaderStyles } from './PageHeader.types';
-import { css, ScreenWidthMinUhfMobile, FontWeights, IStyleFunction, classNamesFunction, styled } from '@fluentui/react';
+import {
+  classNamesFunction,
+  styled,
+  DirectionalHint,
+  FontWeights,
+  IStyleFunction,
+  ScreenWidthMinUhfMobile,
+  IContextualMenuItem,
+} from '@fluentui/react';
+import { ActionButton } from '@fluentui/react/lib/compat/Button';
 import { FontSizes } from '@fluentui/theme';
-import { appPaddingSm, appPaddingLg, pageHeaderFullHeight } from '../../styles/constants';
+import { appPaddingSm, appPaddingLg, contentWidth, pageHeaderFullHeight } from '../../styles/constants';
 
 const getStyles: IStyleFunction<IPageHeaderStyleProps, IPageHeaderStyles> = props => {
   const palette = props.theme!.palette;
   return {
-    root: {
-      position: 'relative',
-      marginBottom: appPaddingSm,
-      selectors: {
-        [`@media screen and (min-width: ${ScreenWidthMinUhfMobile}px)`]: {
-          marginBottom: 0,
-          padding: `${appPaddingLg}px 0`,
-          minHeight: pageHeaderFullHeight,
+    root: [
```

Comment:
Hmm I see what you mean...looking at the master/7 PR deploy site to avoid interference by UHF styling, in the [7.0 version](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/7.0/fabric-website/dist/index.html?devhost#/controls/web/button) it's fine but [in master](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/public-docsite/index.html?devhost#/controls/web/button) there's the extra space. I guess I'll look into it sometime before release.